spring-boot/proxy/src/test/java/LocalApiKeyManagerTest.java

```diff
@@ -44,15 +40,16 @@
     @BeforeEach
     public void setup() throws ApiKeyFetchingException {
         apiKey = new ApiKey();
-        apiKey.setKeyValue(someValidApiKeyValue);
+        apiKey.setKeyValue("ca716b82-745c-4f6d-a38b-ff8fe140ffd1");
 
         localApiKeyList = new ArrayList<>();
         localApiKeyList.add(apiKey);
 
         apiKeyFetcherMock = mock(ApiKeyFetcher.class);
         when(apiKeyFetcherMock.requestLatestApiKeyListFromAdmin()).thenReturn(localApiKeyList);
 
-        localApiKeyManager = new LocalApiKeyManager(apiKeyFetcherMock, oneHourInMinutes);
+        long timeInMinutesTheGatewayWorksWithoutConnectionToAdmin = 60;
```

Comment:
As you already defined the time in the application.yml, couldn't you use `@Value("${number-of-minutes-the-gateway-works-without-connection-to-admin}")` ?

Reply:
@Value is executed when Spring is run which my tests don't do, however.

Reply:
This might help, please check it
https://stackoverflow.com/questions/52293575/value-annotation-not-working-in-junit-test

Reply:
Actually, this parameter is not significant for the test at all. Therefore I see no benefit in this. Setting up a Spring Boot App just seems to cost performance and lines of code. Anyhow, this could be useful for future tests, so thanks for the link.

Reply:
Ok, the idea was to improve the code, but indeed as it needs more than one line of code and extra configuration for Spring Tests, declaring a variable seems okay